sql/expected_result.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sql
+
+// ExpectedResultType represents the expected statement result type.
+type ExpectedResultType byte
+
+const (
+	ANY_RESULT          = ExpectedResultType(0)
+	ROWS_RESULT         = ExpectedResultType(1)
+	UPDATE_COUNT_RESULT = ExpectedResultType(2)
```

Comment:
Let's use upper-camel case for constants like, `AnyResult`.

Reply:
Sure, fixed at https://github.com/hazelcast/hazelcast-go-client/pull/755/commits/39a84bbc27b6aea101f25b0c5dc50ba1bb1175b8